### PR TITLE
Async processing of course uploads.

### LINKED ISSUE #28: Asynchronous Course Import
As a user, I would like to not have my course imports timeout when I either have a slow Internet connection, a large course, or a bit of both.  Dependent on #4

### PR DESCRIPTION
**Pull request by ShawnMilo**
Closes #28.

### CODE REVIEW

File: learningresources/models.py [hunk 1]
```diff
@@ -46,10 +46,6 @@ class LearningResource(models.Model):
     xa_avg_grade = models.FloatField(default=0)
     xa_histogram_grade = models.FloatField(default=0)
 
-    class Meta:
```
Comment (noisecapella) [inline learningresources/models.py:51]: Is this removal intentional? If so I'm fine with it, commenting just in case it's an oversight

File: importer/tasks.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+"""
```
Comment (noisecapella) [inline importer/tasks.py:1]: No from **future** import unicode here? It may be a good idea to add verifying this exists to our list of things to check when tox runs. 

We don't need it here but it would be good to be consistent across the application and to prevent possible oversights for future commits

File: learningresources/models.py (hunk 1 above)
Comment (ShawnMilo) [inline learningresources/models.py:51]: Yes, it was intentional because things were breaking without it. I could have fixed it with a migration (and did, for a while), but since #80 is out there pending, I figured I'd just wait for it to be merged instead of fixing it twice.

Comment (noisecapella) [inline learningresources/models.py:51]: I figured, makes sense

File: importer/tasks.py (hunk 2 above)
Comment (ShawnMilo) [inline importer/tasks.py:1]: It's a good idea to have in general, and it's only not there because I forgot it. There are no strings in this file, though, so it doesn't matter.

File: lore/celery.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+async = Celery('lore')  # pylint: disable=invalid-name
+
+# Using a string here means the worker will not have to
+# pickle the object when using Windows.
+async.config_from_object('django.conf:settings')
+async.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+@async.task(bind=True)
```
Comment (noisecapella) [inline lore/celery.py]: Was this added for local testing? If it's not needed in production we should remove it

Comment (ShawnMilo) [inline lore/celery.py]: Yep, it was copy/paste out of the sample. But I've tested actual imports, so it should go. Good catch.

File: importer/tasks.py (hunk 2 above)
Comment (ShawnMilo) [inline importer/tasks.py:1]: Added (it fell below this comment).

**Comment by noisecapella** [comment]
Looks good, I'll merge when Travis finishes

**Comment by ShawnMilo** [comment]
:exclamation: ^_^

**Comment by noisecapella** [comment]
The decrease in code coverage is from the tests doing a direct upload instead of using celery to do it async. Seems reasonable to me
